_includes/index-episode-pane.njk

```diff
@@ -0,0 +1,25 @@
+<li class="pane index-pane">
+<div class="index-media">
+    <!-- Media object -->
+    <div class="index-media-image">
+        <img src="/media/{{ src.imageFilename }}" class="index-image" alt="Episode image" />
+    </div>
+
+    <!-- Main content -->
+    <div class="index-media-content">
+          <div class="episode-title">
+            <a href="/episodes/{{ src.slug }}/index.html">{{ src | getEpisodeTitle }}</a>
+          </div>
+          <audio
+            class="episode-player"
+            controls=""
+            loop="false"
+            preload="metadata"
+            src="/episodes/rehydrate-s{{ src.season }}-ep{{ src.number }}-{{ src.slug }}.mp3"
+          ></audio>
+          <div class="time-stamp entry-meta">
+            <time datetime="{{ src.date }}">{{ src.date.toDateString() }}</time>
+          </div>
+    </div>
+</div>
```

Comment:
Is it intentional that all of the Spoilercasts can't be seen from the index?

Reply:
originally it was.. but it's probably a good idea to have some reference.. 

added a new section when we have bonus content

![image](https://user-images.githubusercontent.com/26289/110737157-b0df9700-81e1-11eb-889e-f6a2c34b8989.png)

Reply:
Nice. Looks a little cramped on Firefox.

![Screenshot_2021-03-11 Rehydrate](https://user-images.githubusercontent.com/324298/110885514-ffaa3100-82b4-11eb-80a0-6d35bd31f3b9.png)

```
.bonus-section {
    margin-top: 1rem;
}
```
Or something close to that will probably help.

Reply:
I actually ended up messing with it even more

![image](https://user-images.githubusercontent.com/26289/110889247-a46f4300-82a2-11eb-9c64-fcf12bdff90c.png)

![image](https://user-images.githubusercontent.com/26289/110889270-adf8ab00-82a2-11eb-9bb6-66f4be9439bb.png)

Reply:
Nice flexing!